runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -34,273 +34,313 @@
+			keyName = "recoilNotification",
+			name = "Ring of Recoil Notification",
+			description = "Configures if the ring of recoil breaking notification is shown",
+			position = 16
 	)
 	default boolean recoilNotification()
 	{
 		return false;
 	}
 
 	@ConfigItem(
-		keyName = "showBindingNecklaceCharges",
-		name = "Show Binding Necklace Charges",
-		description = "Configures if binding necklace item charge is shown",
-		position = 17
+			keyName = "showBindingNecklaceCharges",
+			name = "Show Binding Necklace Charges",
+			description = "Configures if binding necklace item charge is shown",
+			position = 17
 	)
 	default boolean showBindingNecklaceCharges()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "bindingNecklace",
-		name = "",
-		description = "",
-		hidden = true
+			keyName = "bindingNecklace",
+			name = "",
+			description = "",
+			hidden = true
 	)
 	default int bindingNecklace()
 	{
 		return -1;
 	}
 
 	@ConfigItem(
-		keyName = "bindingNecklace",
-		name = "",
-		description = ""
+			keyName = "bindingNecklace",
+			name = "",
+			description = ""
 	)
 	void bindingNecklace(int bindingNecklace);
 
 	@ConfigItem(
-		keyName = "bindingNotification",
-		name = "Binding Necklace Notification",
-		description = "Configures if the binding necklace breaking notification is shown",
-		position = 18
+			keyName = "bindingNotification",
+			name = "Binding Necklace Notification",
+			description = "Configures if the binding necklace breaking notification is shown",
+			position = 18
 	)
 	default boolean bindingNotification()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showExplorerRingCharges",
-		name = "Show Explorer's Ring Alch Charges",
-		description = "Configures if explorer's ring alchemy charges are shown",
-		position = 19
+			keyName = "showExplorerRingCharges",
+			name = "Show Explorer's Ring Alch Charges",
+			description = "Configures if explorer's ring alchemy charges are shown",
+			position = 19
 	)
 	default boolean showExplorerRingCharges()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "explorerRing",
-		name = "",
-		description = "",
-		hidden = true
+			keyName = "explorerRing",
+			name = "",
+			description = "",
+			hidden = true
 	)
 	default int explorerRing()
 	{
 		return -1;
 	}
 
 	@ConfigItem(
-		keyName = "explorerRing",
-		name = "",
-		description = ""
+			keyName = "explorerRing",
+			name = "",
+			description = ""
 	)
 	void explorerRing(int explorerRing);
 
+	@ConfigItem(
+			keyName = "showRingOfForgingCount",
+			name = "Ring of Forging Count",
+			description = "Configures if the Ring of Forging charge count is shown",
+			position = 20
+	)
+	default boolean showRingOfForgingCount()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "RingOfForgingNotification",
+			name = "Ring of Forging Notification",
+			description = "Configures if the Ring of Forging breaking notification is shown",
+			position = 21
+	)
+	default boolean ringOfForgingNotification()
```

Comment:
Move this notification default method below the `ringOfForging` hidden and setter methods.